src/com/projectkorra/projectkorra/GeneralMethods.java

```diff
@@ -2532,4 +2500,51 @@ public static boolean isLightEmitting(final Material material) {
 				return false;
 		}
 	}
+
+	@Deprecated
+	public static void setVelocity(Entity entity, Vector vector) {

```

Comment:
I deprecated this method... Do you want me to add a nag method to encourage other authors to adopt this new method?   Currently most addon's dont even use the GeneralMethods version so it might be safe for removal entirely.